README.md

```diff
@@ -92,18 +92,20 @@ Inside your `AppDelegate.application(_, handleEventsForBackgroundURLSession:comp
 Once the user has logged into your app, inform Streem that they are logged in:
 
 ```swift
-    Streem.sharedInstance.identify(
-        userId: "john",
-        expert: false,
-        name: "John Smith", 
-        avatarUrl: "http://..."
-        ) { success in
-            if success {
-                // dismiss login screen, etc.
-            } else {
-                // present alert, etc.
-            }
+    Streem.sharedInstance.identify(with: code, avatarUrl: "http://...") { error, details in
```

Comment:
Since this `identify` takes an invitation code, we should introduce it later in the documentation after we have told the reader about invitation codes.

Reply:
👍🏻